packages/block-library/src/table-of-contents/edit.js

```diff
@@ -295,7 +293,7 @@ export default function TableOfContentsEdit( {
 	return (
 		<>
 			<nav { ...blockProps }>
-				<ol ref={ disabledRef }>
+				<ol inert="true">
```

Comment:
As noted here, this introduced an accessibility regression. Please don't do this in the future.

https://github.com/WordPress/gutenberg/pull/54322#discussion_r1320386448